xarray/datatree_/docs/source/conf.py

```diff
@@ -17,9 +17,9 @@
 import os
 import sys
 
-import sphinx_autosummary_accessors
+import sphinx_autosummary_accessors # type: ignore
 
-import datatree
+import datatree # type: ignore
```

Comment:
you shouldn't need that if we're excluding this from type-checking (but no reasons against making the change)

Reply:
I'll leave it.  thanks